packages/expo-sqlite/src/__tests__/hooks-test.ios.tsx

```diff
@@ -9,7 +9,7 @@ jest.mock('../ExpoSQLite', () => require('../__mocks__/ExpoSQLite'));
 
 describe(useSQLiteContext, () => {
   it('should return a SQLite database instance', async () => {
-    const wrapper = ({ children }) => (
+    const wrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
I highly prefer `({ children }: PropsWithChildren)` over typing `{ children: React.ReactNode }` - technically should be `{ children?: React.ReactNode }`.

Reply:
no strong preference here, but fair enough. I've been burned by over-use of `@types/react`-provided types before, and usually now stick with built-in JSX namespace types, with the exception of `React.ReactNode` (or React-specific ones as needed). This also makes a lot of the types as specific as they have to be